src/SiteAlias/AliasRecord.php

```diff
@@ -116,7 +117,9 @@ public function hasRoot()
      */
     public function root()
     {
-        return $this->get('root');
+        $root = FsUtils::realpath($this->get('root'));
```

Comment:
This is unsafe. 'root' might refer to a remote site, and realpath will return `false` if the remote root does not also exist locally. Keep the realpath in the localRoot function.

Reply:
Oh, I see what you did here. 😄